adapters/repos/db/inverted/bm25_searcher.go

```diff
@@ -168,29 +160,26 @@ func (b *BM25Searcher) wand(
 
 	averagePropLength = averagePropLength / float64(len(params.Properties))
 
-	// preallocate the results
-	lengthAllResults := 0
-	for tokenization, propNames := range propNamesByTokenization {
-		if len(propNames) > 0 {
-			lengthAllResults += len(queryTermsByTokenization[tokenization])
-		}
-	}
-	results := make(terms, lengthAllResults)
-	indices := make([]map[uint64]int, lengthAllResults)
+	results := make(terms, 0, 100)
```

Comment:
I'd extract this into a constant called something like, EXPECTED_NUMBER_OF_TERMS

Reply:
I'm a bit dubious about adding a constant when it is only used in one place, would adding a comment help?

Reply:
Adding a comment would help, yes.